src/@next/components/organisms/StripePaymentGateway/StripePaymentGateway.tsx

```diff
@@ -54,7 +65,22 @@ const StripePaymentGateway: React.FC<IProps> = ({
           expYear: card?.exp_year,
           lastDigits: card?.last4,
         });
+      } else {
+        const stripePayloadErrors = [
+          { message: "Stripe returns no payment method in payload." },
```

Comment:
I feel like it should be  "Stripe returned..."

Reply:
yes, sure